project4/style.css

```diff
@@ -0,0 +1,185 @@
+/* == Base styles == */
+* {
+    font-family: Helvetica, sans-serif;
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+}
+
+.hashtag img {
+    height: auto;
+    width: 46px;
+}
+
+.hashtag.twitter img {
+    width: 50px;
+    height: 50px;
+
+
+}
+
+/* == Nav bar == */
+.nav-bar {
+    background-color: #eab5c2;
+    padding-top: 60px;
+    position: relative;
+}
+
+@media (min-width: 900px) {
+    .nav-bar {
+        padding-top: 0;
+    }
+}
+
+/* == Nav list == */
+.nav-items {
+    list-style: none;
+    margin: 0;
+    max-height: 1px;
+    overflow: hidden;
+    padding: 0;
+    transition: all 1s cubic-bezier(0.325, 1, 0.22, 1);
+    opacity: 0.7;
+}
+
+.nav-item {
+    background: #eab5c2;
+    margin: 0;
+    padding: 0;
+}
+
+.nav-item a {
+    display: block;
+    border-bottom: 1px solid white;
+    color: #eab5c2;
+    background: #444;
+    font-size: 1.2rem;
+    margin: 0;
+    padding: 1em;
+    text-decoration: none;
+    transition: opacity 0.5s ease;
+}
+
+.nav-item a:link,
+.nav-item a:visited {
+    color: white;
+}
+
+.nav-item a:hover,
+.nav-item a:focus {
+    opacity: 0.7;
+    text-decoration: none;
+}
+
+@media (min-width: 900px) {
+    .nav-items {
+        max-height: none;
+        text-align: center;
+    }
+
+    .nav-item {
+        display: inline-block;
+    }
+
+    .nav-item a {
+        border: none;
+        display: inline-block;
+        height: 60px;
+        line-height: 60px;
+        padding-top: 0;
+        padding-bottom: 0;
+    }
+}
+
+/* == Hamburger icon == */
+.nav-icon {
+    border: none;
+    color: #eab5c2;
+    cursor: pointer;
+    display: block;
+    height: 27px;
+    position: absolute;
+    left: 13px;
+    text-align: center;
+    text-indent: -9999px;
+    text-shadow: #eab5c2;
```

Comment:
[`text-shadow`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-shadow) requires more values than this to work.